readme.md

```diff
@@ -517,6 +519,12 @@
 - [Blockchain AI](https://github.com/steven2358/awesome-blockchain-ai#readme) - Blockchain projects for artificial intelligence and machine learning.
 - [EOSIO](https://github.com/DanailMinchev/awesome-eosio#readme) - A decentralized operating system supporting industrial-scale apps.
 
+
+## Higher Education
+
+- [Computational Neuroscience](https://github.com/eselkin/awesome-computational-neuroscience) - Computational neuroscience institutions and researchers.
```

Comment:
Couldn't this be in the `Computer Science` category?

Reply:
In most schools it comes under the psychology, engineering, medicine or other departments. It is hardly ever in computer science.

Reply:
Instead of repeating `Computational neuroscience` in the description, try to succinctly describe what `Computational neuroscience` is.

Reply:
Done: https://github.com/sindresorhus/awesome/pull/1585/commits/b157203eb304c134536ec34cabd87fc94c4449a3